src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -52,15 +58,69 @@ private Module[] getModules() {
     }
 
     private void configurePathPlanner() {
-        //        AutoBuilder.configureHolonomic(
-        //                () -> RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose(),
-        //                (pose) -> RobotContainer.POSE_ESTIMATOR.resetPose(AlliancePose2d.fromBlueAlliancePose(pose)),
-        //                this::getSelfRelativeVelocity,
-        //                this::selfRelativeDrive,
-        //                SwerveConstants.HOLONOMIC_PATH_FOLLOWER_CONFIG,
-        //                () -> !DriverStationUtils.isBlueAlliance(),
-        //                this
-        //        );
+        AutoBuilder.configureHolonomic(() -> RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose(),
+                (pose) -> RobotContainer.POSE_ESTIMATOR.resetPose(AlliancePose2d.fromBlueAlliancePose(pose)),
+                this::getSelfRelativeVelocity,
+                this::selfRelativeDrive,
+                SwerveConstants.HOLONOMIC_PATH_FOLLOWER_CONFIG,
+                () -> !DriverStationUtils.isBlueAlliance(),
+                this
+        );
+    }
+
+    public ChassisSpeeds getSelfRelativeVelocity() {
+        return SwerveConstants.KINEMATICS.toChassisSpeeds(getModuleStates());
+    }
+
+    private void selfRelativeDrive(ChassisSpeeds chassisSpeeds) {
+        chassisSpeeds = discretize(chassisSpeeds);
+        if (isStill(chassisSpeeds)) {
```

Comment:
consider changing the name of `isStill` to something like `isStableAt` because `isStill` sounds like not moving instead of not accelerating.